apps/ftu/style/style.css

```diff
@@ -368,12 +368,14 @@ gaia-buttons.forward-only .icon-previous {
 #wifi [data-type="list"] li > p {
   pointer-events: none;
   -moz-padding-end: 6.2rem;
+  line-height: 1.33;
```

Comment:
I just wanted to restore the initial default line-height. Maybe unset would have been a better choice?